frontend/www/js/omegaup/components/common/Navbarv2.vue

```diff
@@ -191,55 +190,76 @@
                   <a class="dropdown-item" href="/badge/list/">{{
                     T.navViewBadges
                   }}</a>
-                  <a class="dropdown-item" :href="problemsUrl">{{
-                    problemsText
-                  }}</a>
-                  <a
-                    class="dropdown-item"
-                    :href="coursesUrl"
-                    data-nav-courses-mine
-                    >{{ coursesText }}</a
-                  >
-                  <a
-                    class="dropdown-item"
-                    :href="contestsUrl"
-                    data-nav-user-contests
-                    >{{ contestsText }}</a
-                  >
-                  <a
-                    v-if="hasTeachingObjective"
-                    class="dropdown-item"
-                    href="/profile/#created-content"
-                    >{{ T.navMyContent }}</a
-                  >
-                  <div v-else class="btn-group dropdown-submenu">
+                  <div v-if="hasTeachingObjective">
```

Comment:
los `<div>` sólo se deben usar cuando es necesario introducir un elemento en el DOM. como aquí lo único que se necesita es agrupar los elementos, se puede usar [`<template v-if>`](https://vuejs.org/v2/guide/conditional.html#Conditional-Groups-with-v-if-on-lt-template-gt), que Vue elimina completamente del DOM.

```suggestion
                  <template v-if="hasTeachingObjective">
```